### PR TITLE
chore(curriculum): add product landing page lab

### PR DESCRIPTION
**Pull request by zairahira**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

**Comment by moT01** [comment]
Blocked while we figure out if we want to use new challenge ID's or the existing ones.

**Review by moT01 — CHANGES_REQUESTED** [review]
We should use the ID of the existing landing page project here - `587d78af367417b2b2512b04`

**Review by jdwilkin4 — COMMENTED** [review]
Remember we discussed using the final solution from the live demo project.
so the HTML and CSS solutions will need to be updated

**Comment by zairahira** [comment]
Thanks for pointing that out @jdwilkin4.
I have updated the HTML/ CSS.

image

File: curriculum/challenges/english/25-front-end-development/lab-product-landing-page/66d9639a8db48f247aa643d2.md [hunk 1]
```diff
@@ -0,0 +1,734 @@
...
+const el = document.getElementById('header-img')
+assert(!!el && !!el.src)
+```
+
+Your `#header-img`’s `src` value should be a valid URL (starts with `http`).
+
+```js
+const el = document.getElementById('header-img')
+assert(!!el && /^http/.test(el.src))
+```
```
Comment (Sembauke) [inline curriculum/challenges/english/25-front-end-development/lab-product-landing-page/66d9639a8db48f247aa643d2.md:68]: I am able to pass this test without adding something in the src.

**Review by Sembauke — APPROVED** [review]

File: curriculum/challenges/english/25-front-end-development/lab-product-landing-page/66d9639a8db48f247aa643d2.md [hunk 2]
```diff
@@ -0,0 +1,734 @@
...
+
+**User Stories:**
+
+1. Your product landing page should have a `header` element with a corresponding `id="header"`
+2. You should have an image within the `header` element with a corresponding `id="header-img"` (A logo would make a good image here)
+3. Within the `#header` element, you should have a `nav` element with a corresponding `id="nav-bar"`
+4. You should have at least three clickable elements inside the `nav` element, each with the class `nav-link`
+5. When you click a `.nav-link` button in the `nav` element, you should be taken to the corresponding section of the landing page
+6. You should have an embedded product video with `id="video"`
+7. Your landing page should have a `form` element with a corresponding `id="form"`
```
Comment (Sembauke) [inline curriculum/challenges/english/25-front-end-development/lab-product-landing-page/66d9639a8db48f247aa643d2.md:21]:
```suggestion
7. Your landing page should have a `form` element with a corresponding `id="form"`.
```

**Review by Sembauke — CHANGES_REQUESTED** [review]
Sorry I approved too quickly.

File: curriculum/challenges/english/25-front-end-development/lab-product-landing-page/66d9639a8db48f247aa643d2.md (hunk 1 above)
Comment (zairahira) [reply curriculum/challenges/english/25-front-end-development/lab-product-landing-page/66d9639a8db48f247aa643d2.md]: It's fixed now.

**Review by jdwilkin4 — APPROVED** [review]

**Review by moT01 — CHANGES_REQUESTED** [review]
The challenge filename should change to match the challenge ID.

**Review by moT01 — APPROVED** [review]
LGTM 🎉 

My only suggestion would be to change that lorem ipsum to more realistic content - but it was already like that on the existing project, so I'm fine with it for now 👍 Feel free to change it though if you want @zairahira

**Comment by zairahira** [comment]
I have updated the `lorem` text 👍️

**Review by Sembauke — APPROVED** [review]

**Pull request merged**